tools/BUILD.bazel

```
@@ -13,6 +14,21 @@ ts_config(
     deps = ["tsconfig.json"],
 )
 
+rules_js_tsconfig(
+    name = "tsconfig",
+    src = "tsconfig.json",
+    deps = ["//:node_modules/@types/node"],
+)
+
+rules_js_tsconfig(
+    name = "tsconfig_test",
```

Can we use the naming convention we used in components

```suggestion
    name = "test-tsconfig",
```

---

I want to switch it the other way around. The target should start with `tsconfig` and follow the name of the config more closely.

---

I would like for us to have `build` or `test` included in the naming to make it more obvious.  I would be fine with even having the underlying `tsconfig.json` files named that way as well.

---

I'm good adding `_build` suffix. The tsconfig itself should be `tsconfig.json` so that it's picked up by IDEs as well.

My goal is to fix that we have IDE tsconfigs and build configs. Although I believe we need a general post-cleanup on this as well, and could follow a pattern similar to the CLI.

---

So just to make sure, as I am having to set up the same for adev/shared-docs right now.

`tsconfig.json` <-> `tsconfig_build`

---

Yes, that seems good to me. Personally I'd prefer dropping `_build` because the build config may be used more often, but the suffix is the compromise.